pkg/graph/resolvers/system_intake_relation.go

```diff
@@ -153,6 +153,8 @@ func UnlinkSystemIntakeRelation(
 		intake.ContractName = zero.StringFromPtr(nil)
 
 		// Clear contract number relationships by setting an empty array of contract #'s
+		// declare this as an explicit empty slice instead of `nil`
+		// TODO: (Sam) update `SetSystemIntakeContractNumbers` to allow for `nil`
```

Comment:
Is this just a semantic change where you'd like to handle interpreting `nil` as "delete all" in the storage method?